pkgs/build-support/rust/replace-workspace-values.py

```diff
@@ -102,21 +111,15 @@ def main() -> None:
 
     changed |= replace_dependencies(workspace_manifest, crate_manifest)
 
-    if "target" in crate_manifest:
-        for key in crate_manifest["target"].keys():
-            changed |= replace_dependencies(
-                workspace_manifest, crate_manifest["target"][key]
-            )
-
-    if (
-        "lints" in crate_manifest
-        and "workspace" in crate_manifest["lints"]
-        and crate_manifest["lints"]["workspace"] is True
-    ):
+    for value in crate_manifest.get("target", {}).values():
+        changed |= replace_dependencies(workspace_manifest, value)
+
+    if crate_manifest.get("lints", {}).get("workspace") is True:
+        changed = True
         crate_manifest["lints"] = workspace_manifest["lints"]
 
     if not changed:
-        return
+        return print(f"{sys.argv[1]} is unchanged, skipping", file=sys.stderr)
```

Comment:
```suggestion
        print(f"{sys.argv[1]} is unchanged, skipping", file=sys.stderr)
        return
```

Reply:
`return print(...)` is basically "exit with a message", semantically it's a single statement